.ipynb_checkpoints/LoRA_Inference-checkpoint.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e049e6ff-2710-425e-9c7c-e42fe0c04b57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_NAME_FOR_INFERENCE=\"${zainhas/Meta-Llama-3.1-8B-Instruct-Reference-my-demo-finetune-4224205a}\"\n",
+    "\n",
+    "curl -X POST https://api.together.xyz/v1/completions \\\n",
+    "  -H \"Authorization: Bearer $TOGETHER_API_KEY\" \\\n",
+    "  -H \"Content-Type: application/json\" \\\n",
+    "  -d '{\n",
+    "    \"model\": \"'$MODEL_NAME_FOR_INFERENCE'\",\n",
+    "    \"messages\": [\n",
+    "      {\n",
+    "        \"role\": \"user\",\n",
+    "        \"content\": \"debate the pros and cons of AI\",\n",
+    "      },\n",
+    "    ],\n",
+    "    \"max_tokens\": 128\n",
+    "  }'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7822bc8-c7c6-47cf-a3d4-9b0823d30fce",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "11d802b7e0038ef5c5cb7768d5e62e8416666a6e998361bba7a4db3031762f1c"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "f8020959-3615-4f20-8065-a185742db068",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'id': '8f2cb236c80ea20e-YYZ', 'object': 'text.completion', 'created': 1734331375, 'model': 'zainhas/Meta-Llama-3.1-8B-Instruct-Reference-my-demo-finetune-4224205a', 'prompt': [], 'choices': [{'text': \"Here's a debate on the pros and cons of AI:\\n\\n**Moderator:** Welcome to today's debate on the pros and cons of AI. We have two debaters, Alex and Ben, who will present their arguments on the topic. Alex will argue in favor of AI, while Ben will argue against it. Let's begin with opening statements.\\n\\n**Alex (In Favor of AI):** Thank you, Moderator. AI has revolutionized the way we live and work. It has improved efficiency, productivity, and accuracy in various industries, such as healthcare, finance, and transportation. AI-powered systems can analyze vast amounts of data, identify\", 'finish_reason': 'length', 'seed': 5626645655383684000, 'logprobs': None, 'index': 0}], 'usage': {'prompt_tokens': 18, 'completion_tokens': 128, 'total_tokens': 146, 'cache_hit_rate': 0}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "api_key = \"11d802b7e0038ef5c5cb7768d5e62e8416666a6e998361bba7a4db3031762f1c\"\n",
```

Comment:
Are you sure you want to have your API key here presented?

Reply:
was mistake - I have regenerated a new key